pkg/services/tracer/jaeger.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tracer
+
+import (
+	"errors"
+	"io"
+	"time"
+
+	"github.com/opentracing/opentracing-go/ext"
+
+	"github.com/uber/jaeger-client-go"
+
+	jaegerPrometheus "github.com/uber/jaeger-lib/metrics/prometheus"
+
+	"github.com/ZupIT/horusec-devkit/pkg/services/tracer/enums"
+
+	"github.com/ZupIT/horusec-devkit/pkg/utils/env"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/uber/jaeger-client-go/config"
+	"github.com/uber/jaeger-client-go/zipkin"
+)
+
+type (
+	Jaeger struct {
+		Name     string
+		LogError bool
+		LogInfo  bool
+	}
+)
+
+func NewJaeger() (*Jaeger, error) {
+	j := &Jaeger{
```

Comment:
The value of `HORUSEC_JAEGER_NAME` is already configurable by `JAEGER_SERVICE_NAME`